### ISSUE TITLE
REFL019 when array

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/ReflectionAnalyzers#208]
```cs
            if (type.IsArray)
            {
                var constructor = type.GetConstructor(new[] { typeof(int) });
```

Should not warn here.

**Comment by JohanLarsson** [comment]
No bug.
